io/net/infiniband/dapl.sh.data/dapl.yaml

```diff
@@ -0,0 +1,16 @@
+!mux
+first:
+    HOST_PARAM: -T S -D $DAPL_IF1
+    PEER_PARAM: -T Q -s $PEER_IP -D $DAPL_IF2 -d
+second:
+    HOST_PARAM: -T S -D $DAPL_IF1 -d
+    PEER_PARAM: -T T -d -s $PEER_IP -D $DAPL_IF2 client SR 4096 2 server SR 4096 2
+third:
+    HOST_PARAM: -T S -D $DAPL_IF1
+    PEER_PARAM: -T  T -V -d -t 2 -w 4 -i 5555 -s $PEER_IP -D $DAPL_IF2 client RW 4096 1 server RW 2048 4 client SR 1024 4 server SR 4096 2 client SR 1024 3 -f server SR 2048 1 -f
+fourth:
+    HOST_PARAM: -T S -D $DAPL_IF1
+    PEER_PARAM: -T P -d -s $PEER_IP -D $DAPL_IF2 -i 100 RW 4096 2 
+fifth:
+    HOST_PARAM: -T L -D $DAPL_IF1 -d -w 16 -m 1000
+    PEER_PARAM: 
```

Comment:
Is the `PEER_PARAM` supposed to be empty here? If yes, you can simple omit it. The presence of it will make avocado consider it a branch and not a leaf:
~~~
$ avocado multiplex --tree io/net/infiniband/dapl.sh.data/dapl.yaml
 ┗━━ run
      ╠══ first
      ╠══ second
      ╠══ third
      ╠══ fourth
      ╚══ fifth
           ┗━━ PEER_PARAM
~~~
No major issues in this case, but the variant naming will be a little misleading:
~~~
$ avocado multiplex io/net/infiniband/dapl.sh.data/dapl.yaml
Variants generated:
Variant 1:    /run/first
Variant 2:    /run/second
Variant 3:    /run/third
Variant 4:    /run/fourth
Variant 5:    /run/fifth/PEER_PARAM
~~~
So you could both remove the line or attribute an empty string for it (`PEER_PARAM: ""`)